apps/documentation/src/content/docs/getting-started.md

```
@@ -0,0 +1,414 @@
+- `search_history` (search logs)
+
+### 5. Start Development Servers
+
+#### Run All Services Concurrently:
+
+```bash
+bun run dev
+```
+
+This uses Turborepo to start:
+- **API Server** at http://localhost:3030
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)